rest/results-tabulation-api/migrations/versions/5f61093c9695_.py

```diff
@@ -0,0 +1,108 @@
+"""empty message
+
+Revision ID: 5f61093c9695
+Revises: ab9d4f52637d
+Create Date: 2020-03-03 21:29:02.871678
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '5f61093c9695'
+down_revision = 'ab9d4f52637d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('election', 'isListed',
```

Comment:
Due to a bug in the migration file generation, some of this existing changes are re populated. Shall we remove all unnecessary changes

Reply:
Fixed.